public/catalogue-frontend.css

```diff
@@ -3163,7 +3163,7 @@ tbody.collapse.in {
   position: absolute;
   top: 100%;
   left: 0;
-  z-index: 1000;
+  z-index: 100000;
```

Comment:
needed to increase this so the dropdown menu items sit on top of the sticky columns